src/utils/generateRandomColor.ts

```diff
@@ -0,0 +1,11 @@
+import randomSeed from 'random-seed';
+
+export function generateRandomColor(seed: string): string {
+    const rand = randomSeed.create(seed);
+    const r = rand(256);
+    const g = 0; //temporarily limit the color range so it looks good
```

Comment:
Oh I'm sorry for being unstructured, but here's one more comment. I think you should put a `TODO` comment here so it's less likely that we forget it.